### DISCUSSION TITLE
WebGIS6 - Drucken von Redlining-Texten

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#213] [Q&A]
Hallo Jürgen!

Ein User hat beim Drucken von Redlining-Texten im WebGIS6 (6.24.4201) ein Problem gefunden.

Er hat 4x das Kürzel "ME" via Redlining-Text positioniert. In der Druckvorschau wird der Text auch angezeigt (siehe Screenshot). 
![image](https://github.com/user-attachments/assets/5b75b720-82c8-4258-a518-48a7405d647d)

Im Ausdruck wird das Kürzel "ME" dann allerdings nur noch 1x angezeigt (siehe nachfolgenden Screenshot).
![image](https://github.com/user-attachments/assets/01ef6424-e82c-4049-b102-272fe2e90d0a)

Was kann denn die Ursache für dieses Verhalten sein? Könnte es ein Platzproblem sein (der Text wird im Ausdruck deutlich größer als in der Vorschau)? Gibt es eine Möglichkeit die Vorschau und den Ausdruck gleich darzustellen bzw. im Ausdruck alle Elemente beizubehalten?

Danke!

Lg, Andi

**Comment by gis-eni** [comment]
Hallo Andi,

ja hier handelt es sich um einem Bug. Das Problem tritt auf, wenn man Text mit immer gleicher Beschriftung zeichnet. Dann wird der vorherige Text zwar in der Karte dargestellt, allerdings dann schlussendlich nicht übernommen:

<img width="373" alt="image" src="https://github.com/user-attachments/assets/675e495b-faef-4ece-bf0b-53edaed34ce4">

Das Verhalten ist natürlich verwirrend und ist dann im nächsten Build behoben.

PS: es ist keine Platzproblem. Der Text ist nicht wirklich größer im Ausdruck. Das wirkt nur so, weil in der "Vorschau" der Druck natürlich in einem anderen Maßstab angezeigt wird. Tatsächlich sind die Text genau gleich groß ;)
